perf/benchmark/configs/istio/none/installation.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operator.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    telemetry:
+      enabled: false
```

Comment:
this is not necessary, the other disable should cover it.

Reply:
ok I would remove it, originally added it because want to blow away the old mixer resources stuff hanging, trying to reduce variance